src/io/stun.rs

```diff
@@ -316,7 +319,7 @@ impl<'a> StunMessage<'a> {
 const MAGIC: &[u8] = &[0x21, 0x12, 0xA4, 0x42];
 
 #[derive(Clone, Copy, Debug, PartialEq)]
-enum Class {
+pub(crate) enum Class {
```

Comment:
I believe that with the way things are set up, these enums are now visible to other modules in str0m, but not to consumers of the crate, so public API remains unchanged. Let me know if there's a better way to do this, though.

Reply:
pub(crate) is good